src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/TypeAccessibilityCodeActionProvider.cs

```diff
@@ -41,22 +41,12 @@ internal sealed class TypeAccessibilityCodeActionProvider : ICSharpCodeActionPro
         IEnumerable<RazorVSInternalCodeAction> codeActions,
         CancellationToken cancellationToken)
     {
-        if (context is null)
-        {
-            throw new ArgumentNullException(nameof(context));
-        }
-
-        if (codeActions is null)
-        {
-            throw new ArgumentNullException(nameof(codeActions));
-        }
-
         if (context.Request?.Context?.Diagnostics is null)
         {
             return SpecializedTasks.AsNullable(SpecializedTasks.EmptyReadOnlyList<RazorVSInternalCodeAction>());
         }
 
-        if (codeActions is null || !codeActions.Any())
+        if (!codeActions.Any())
```

Comment:
Can we switch this type to a concrete type to avoid using `Any()`?

Reply:
Was avoiding changing the interface to avoid conflicts etc. maybe a follow up in the feature branch once this merges up?